### PR TITLE
Revert "Change default block priority size to 0"

### PR DESCRIPTION
**Pull request by ghost**
We should restore free relay so that these transactions will confirm properly.

### CODE REVIEW

**Comment by Aquentus** [comment]
Welcome to github. Is there any specific reason why you are making these pull requests under a 0day rather than your real nickname? Considering that you seem very knowledgable and therefore are probably a known contributor, it raises suspicions that you are up to no good when you choose to make such pull requests under "cointracker" rather than have the courage to put your face next to your code.

Anyway, Bitcoin Classic has a certain process which I can not fault you for not following. You are first to propose a BCIP in the appropriate forum, but, you can bypass that for now by putting up your suggestion on consider.it for a vote. If the community is in favour then you can pull request the code which I would hope you have thoroughly tested. Until then I would recommend that you refrain from spamming this github with some 6 pull requests in a matter of hours as your real identity is quite obvious to all.

**Comment by jtoomim** [comment]
@Aquentus One hypothesis is that there may be drama at his employer. I suggest not digging too deeply.

**Comment by jtoomim** [comment]
These pull requests are known issues with work that Core has merged into their master branch lately, and we will want to review them carefully to decide which to revert. @cointracker is doing work that needs to be done eventually. 

So far, all or almost of this will probably sit in open pull requests until after the hard fork releases are done, but that's okay. cointracker wants to enumerate them now, and I like to see things get done.

**Comment by Aquentus** [comment]
Fair enough (although some of his comments seem pretty trollish). 

I don't necessarily disagree with the concept of the pull request, but this pull request and the others will have to be debated/discussed first in an appropriate forum. As, although I do lean towards a concept ACK for this pull request, I am sure there are things to be said and considered both in favour and against. 

Ah, thinking about it, we do have https://bitco.in/forum/forums/bitcoin-classic.20/ where indepth discussions can be had and we'll prob want a mailing list as well in due course.

File: src/policy/policy.h [hunk 1]
```diff
@@ -18,7 +18,7 @@ class CCoinsViewCache;
 static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 750000;
 static const unsigned int DEFAULT_BLOCK_MIN_SIZE = 0;
 /** Default for -blockprioritysize, maximum space for zero/low-fee transactions **/
-static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 0;
+static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 50000;
```
Comment (orangeman) [inline src/policy/policy.h:21]: This is a serious issue and needs to be addressed before release.

Please stop jamming the technical discussion @Aquentus!

**Comment by ghost** [comment]
@Aquentus I am not a contributor but I am familiar with the bitcoin code base as we run many heavily modified clients that attempt to make many connections with other nodes on the bitcoin network for double spend and compliance tracking. Our services are strictly business to business for companies we have prior relationships with and we don't advertise publicly at all.

> You are first to propose a BCIP in the appropriate forum, but, you can bypass that for now by putting up your suggestion on consider.it for a vote.

I thought a BCIP is only for consensus or protocol changes.

@jtoomim No drama at all, we just prefer not to have our identities public, this is common in our industry.

**Comment by jtoomim** [comment]
Did the CreateNewBlock code that @morcos wrote leave the priority space in a usable state? I was under the impression that they chose not to implement it at the time due to the way they were doing mempool sorting by fee.

Edit: almost right, the problem is with the eviction code.

**Comment by ghost** [comment]
@jtoomim I'm not sure but maybe that would need to be reverted if it is the case.

**Comment by jtoomim** [comment]
@cointracker Can you provide some data showing that this PR actually works, and that it changes how blocks are assembled correctly?

**Comment by sugruedes** [comment]
Can miners not set -blockprioritysize=50000 to reset to 50k if they so wish as a local policy decision?

If so,then  this is not a showstopper, and should be postponed for a later release, as part of a policy to retain the concept of 'priority'

**Comment by jtoomim** [comment]
https://github.com/bitcoin/bitcoin/pull/7022

> As discussed on the mailing list, the current notion of priority is no longer completely supported with the 0.12 release. (In particular limited mempools will not allow free transactions when they have recently hit their limit). It is likely that further support will be removed in future release and the current concept is now deprecated. As such it makes sense to make the mining default not include priority space. It is still supported to create a priority space if the command line argument is changed. (To what degree depends on other open PR's)

**Pull request closed without merging**